task_api/backends/celery.py

```diff
@@ -2,7 +2,10 @@
 
 import json
 
-from celery import task
+try:
+    from celery import task
+except ModuleNotFoundError:
+    from celery import shared_task as task
```

Comment:
This doesn't work as expected, because `celery.task` still exists, it's just a module now rather than a decorator. So it will still import, but is not usable.

The good news is that I've confirmed that `shared_task` works celery versions 4 & 5. So if you can revert this to your original change, we can get it merged and I'll release a new version.

Reply:
thx for the testing, I'm not sure if this change is actually needed as long as you have the dependency on celery4 set, I broke my dependencies in my environment by playing with a pip upgrade-all command, in normal use with correct dependencies this error should not have occurred. 
But you might want to keep this pr open in case you consider starting to support celery 5 ?

Reply:
I just noticed celery 4 is no longer supported? https://docs.celeryproject.org/en/stable/history/whatsnew-5.0.html states: 
>  Celery 4.x will be supported until the 1st of August, 2021.